From: Marc Kleine-Budde <mkl@pengutronix.de>
Date: Sun, 27 Mar 2011 15:18:27 +0200
Subject: [PATCH] Fix screenencodings dir

This patch fixes the assignement of the SCREENENCODINGS variable which was holding "${}" before.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 configure.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.in b/configure.in
index 419279d..5411add 100644
--- a/configure.in
+++ b/configure.in
@@ -637,7 +637,7 @@ AC_MSG_RESULT($ETCSCREENRC)
 SCREENENCODINGS="\"/usr/local/lib/screen/utf8encodings\""
 if test -n "$datadir"; then
 eval SCREENENCODINGS="$datadir/screen/utf8encodings"
-SCREENENCODINGS="\"$SCREENENCODINGS\""
+eval SCREENENCODINGS="\\\"$SCREENENCODINGS\\\""
 fi
 AC_MSG_CHECKING(for the utf8-encodings location)
 AC_DEFINE_UNQUOTED(SCREENENCODINGS,$SCREENENCODINGS)
-- 
1.7.4.1

